src/parser.jl

```diff
@@ -130,7 +130,12 @@ end
 curline(ts::TokenStream)  = ts.lineno
 filename(ts::TokenStream) = symbol("none")
 
-line_number_node(ts) = LineNumberNode(curline(ts))
+if VERSION < v"0.4-dev+7034"
+    line_number_node(f, n) = LineNumberNode(n)
```

Comment:
It's in the desctiption, see: https://github.com/MichaelHatherly/Lexicon.jl/issues/131 & https://github.com/MichaelHatherly/Docile.jl/commit/a2509fb62c58409ae8382cb25d7363a2d75118b3